WooCommerce/Classes/POS/Utils/POSFontStyle.swift

```diff
@@ -5,35 +5,32 @@ import SwiftUI
 enum POSFontStyle {
     case posTitleRegular
```

Comment:
Aligning on font style naming with figma

```suggestion
    case posHeadingRegular
```